index.html

```diff
@@ -4,134 +4,162 @@
         <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700" rel="stylesheet">
-        
         <!-- adding CSS for bootstrap-->
         <link rel="stylesheet" href="css/bootstrap.min.css" crossorigin="anonymous">
-        
         <!-- adding CSS for the page-->
         <link rel="stylesheet" href="/css/style.min.css" type="text/css" />
         <title>GameDecision - Find out if you should buy that game</title>
     </head>
     
-    <body ngjq ng-app="gameSearch" ng-controller="gameSearchController" >
+    <body ngjq ng-app="gameJudgement"  >
         <div>
             <!--Google Adsense code-->
             <script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
             <!-- Responsive Ads -->
             <ins class="adsbygoogle responsive_ads"
                  style="display:inline-block"
-                 data-ad-client="MY_ADSENSE_PUB_ID"
-                 data-ad-slot="MY_ADSENSE_SLOT_ID"
+                 data-ad-client="MY_ADSENSE_AD_CLIENT_KEY"
+                 data-ad-slot="MY_ADSENSE_AD_SLOT_KEY"
                  data-ad-format="auto"></ins>
             <script>
             (adsbygoogle = window.adsbygoogle || []).push({});
             </script>
         </div>
-        <div class="container-fluid"> 
-        <h1 id="site-title"><a href="/"><b>GameDecision</b></a></h1>
-        <p id="slogan">Find out if you should buy that game</p>
-        
-        <p>Notes: For recent releases, leave the name box blank, type in a threshold and click "recent releases".</p>
-        <br />
-        <p>Smartphone users: turn your phone sideways/to a landscape orientation.</p>
-        <br />
-        <p>Threshold is the target score that shall be used to determine if a game is worth buying or not. If you decide not to set the threshold then by default it's set to 75.</p>
-        <form class="form-horizontal">
-            <!--A form for the user to type in their search query for their game-->
-            <div class="form-group">
-                <label>Game name:</label><br /><input id="search" type="text" placeholder="game" name="search"/>
-            </div>
-            <div class="form-group">
-                <!--This is the target score which is used to determine if a game is worth buying(which is 75 by default). More on this down below-->
-                
-                <label>Threshold (between 1 and 100 please):</label><br /><input id="target" type="text" placeholder="target" name="target"/>
+        <div ng-controller="gameSearchController">
+            <div ng-controller="recentReleasesCtrl as ctrl">
+                <div class="container-fluid" > 
+                    <h1 id="site-title"><a href="/"><b>GameDecision</b></a></h1>
+                    <p id="slogan">Find out if you should buy that game</p>
+                    <p>Notes: For recent releases, leave the name box blank, type in a threshold and click "recent releases".</p>
+                    <br />
```

Comment:
its not a big deal but it is usually considered bad practice to break up your html using line breaks like this, instead you should style using css so the bad looks the way you want